server/src/deno.ts

```diff
@@ -41,8 +41,11 @@ class Deno {
       );
     }
   }
-  public async getTypes(): Promise<Buffer> {
-    const { stdout } = await execa(this.executablePath as string, ["types"]);
+  public async getTypes(unstable: boolean): Promise<Buffer> {
+    const { stdout } = await execa(this.executablePath as string, [
+      "types",
+      ...(unstable ? ["--unstable"] : []),
```

Comment:
Because there is no `--unstable` flag in the old version, this will cause `deno types --unstable` panic

So here you need to compare the versions and make a compatible